src/wmnds/components/content-card/_content-card.njk

```diff
@@ -3,9 +3,10 @@
 {% set classes = " " + params.classes if params.classes else "" %}
 {% set yellowModifier = " wmnds-content-card--yellow" if params.yellow else "" %}
 {# Opening & closing tags #}
-{% set tag = "a" if params.href else "div" %}
-{% set href = params.href if params.href else "#" %}
-{% set _attributes = " href=\"" + href + "\" target=\"_self\"" if params.href else "" %}
+{% set tag = "a" if params.link else "div" %}
```

Comment:
```suggestion
{% set _tag = "a" if params.link else "div" %}
```

This is just an internal var based on a param, so underscore and drop next to _attributes below.